appium-cookies/test/cookies-specs.js

```diff
@@ -0,0 +1,78 @@
+// transpile:mocha
+
+import cookie from '../lib/cookies.js';
```

Comment:
since `cookie` here is a class we should use the convention of UpperCamelCase for it, i.e., we should import it as `Cookie` not `cookie`.